build/tools/wxwidgets.iss

```diff
@@ -51,7 +51,7 @@ Filename: "{app}\wx.url"; Section: "InternetShortcut"; Key: "URL"; String: "http
 
 [Icons]
 Name: "{group}\{cm:ProgramOnTheWeb,wxWidgets}"; Filename: "{app}\wx.url"
-Name: {group}\wxWidgets Manual; Filename: {app}\docs\htmlhelp\wx.chm; WorkingDir: {app}; IconIndex: 0; Flags: useapppaths
+Name: {group}\wxWidgets Manual; Filename: {app}\docs\htmlhelp\wxWidgets-{#WXW_VER}-docs.chm; WorkingDir: {app}; IconIndex: 0; Flags: useapppaths

```

Comment:
I'm not sure if anyone is relying on this file location being `wx.chm`, I guess I could switch this back though. It just felt weird not explicitly naming the file after the version it represents.